README.md

```diff
@@ -249,9 +249,14 @@ An accelerometer doesn't seem to be available at all.
 
 ## Thunderbolt
 
-![MacBookPro13,1 unknown](https://img.shields.io/badge/MacBookPro13%2C1-unknown-lightgrey.svg) ![MacBookPro13,2 unknown](https://img.shields.io/badge/MacBookPro13%2C2-unknown-lightgrey.svg) ![MacBookPro13,3 unknown](https://img.shields.io/badge/MacBookPro13%2C3-unknown-lightgrey.svg) ![MacBookPro14,1 unknown](https://img.shields.io/badge/MacBookPro14%2C1-unknown-lightgrey.svg) ![MacBookPro14,2 unknown](https://img.shields.io/badge/MacBookPro14%2C2-unknown-lightgrey.svg) ![MacBookPro14,3 unknown](https://img.shields.io/badge/MacBookPro14%2C3-unknown-lightgrey.svg)
+![all models working](https://img.shields.io/badge/all_models-working-green.svg)
```

Comment:
Can you please keep the list of all models here? The general idea is to have something like "all models working" in the overview, but a list of models in each section, to make it clear which models are working.